### DISCUSSION TITLE
Extend hotkey functionality

### DISCUSSION THREAD

**Discussion by andrew-jk-1** [hoothin/UserScripts#714] [Ideas]
For multilingual users as myself, it would be cool to be able to save an image using a hotkey without having to switch to the English keyboard layout every time.

So here is the idea:
Make it possible to set a few hotkeys for the same actions.
OR
Make it possible to set a key codes instead of just key name.
OR 
Make script to recognize and set key codes automatically after user set a hotkey.

Picviewer CE+

**Comment by hoothin** [comment]
![image](https://github.com/user-attachments/assets/1fbe510c-e880-429e-b73a-59c3944a3ebd)
